client-v2/src/components/inputs/richtext/command-helpers.ts

```diff
@@ -0,0 +1,149 @@
+import { parseURL, linkValidator } from './link-validator';
+import { MarkType } from 'prosemirror-model';
+import { EditorState, Transaction } from 'prosemirror-state';
+
+// These prosemirror-helper functions are a simplified version of what we use in Composer, and have been lifted and shifted from that repo
```

Comment:
This is useful to know -- possibly we could think about a shared repo for these things, as PM ends up in more places we'll need these things again and again